src/gsi/read_dbz_nc.f90

```diff
@@ -337,6 +338,7 @@ subroutine read_dbz_nc(nread,ndata,nodata,infile,lunout,obstype,sis,hgtl_full,no
 
   call w3fs21(iadate,mins_an)  !mins_an -integer number of mins snce 01/01/1978
   rmins_an=mins_an             !convert to real number
+  timeb=real(mins_an-iwinbgn,r_kind)
```

Comment:
Sure. this should work for various time-related situations.

Reply:
In other places, timeb is defined by (observation time) - (head of assimilation window) for 4D DA. In case (observation time) = (analysis time) is assumed, 'timeb=real(mins_an-iwinbgn,r_kind)' is required. In the case of 'timeb=zero', GSI regards that reflectivity was observed at the head of the assimilation window, but we don't expect that.

Reply:
In read_dbz_nc.f90,  timeb is commented as "observation time relative to analysis time" and is used as it as in 
```
 cdata_all(7,iout) = timeb*r60inv                  ! obs time (analyis relative hour)
 ```
 So, please stick to this definition of timeb.   If this sub couldn't have any information about actual observation time, just let timeb=0 for both 4dvar/fgat and 3dvar.